src/components/forms/TextField/TextField.js

```diff
@@ -0,0 +1,47 @@
+import {
+  FormFieldDescription,
+  FormLabel,
+  Paragraph,
+  Textbox,
+  FormField as UtrechtFormField,
+} from '@utrecht/component-library-react';
+import classNames from 'classnames';
+import React, {useRef} from 'react';
+
+import {getBEMClassName} from '../../../utils';
+
+export const TextField = ({
+  label = '',
+  isRequired = false,
+  description = '',
+  id = '',
+  disabled,
+  invalid,
+}) => {
+  const labelClassName = classNames(getBEMClassName('label'), {'field-required': isRequired});
+
+  const inputRef = useRef(null);
+
+  const handleLabelClick = () => {
+    inputRef.current.focus();
+  };
+
+  const inputProps = {
+    id,
+    disabled,
+    invalid,
+    ref: inputRef,
+  };
+
+  return (
+    <UtrechtFormField type={'text'} invalid={invalid}>
+      <Paragraph className={labelClassName} onClick={handleLabelClick}>
+        <FormLabel htmlFor={id}>{label}</FormLabel>
+      </Paragraph>
+      <Paragraph>
+        <Textbox {...inputProps} />
+      </Paragraph>
+      {description && <FormFieldDescription invalid={invalid}>{description}</FormFieldDescription>}
+    </UtrechtFormField>
+  );
```

Comment:
We do want to wire this up with Formik directly, and I think something along these lines should be possible:

```jsx
import {Field} from 'formik';

<Paragraph>
  <Field name={name} as={Textbox} {...inputProps} />
</Paragraph>
```

-> the `name` prop should be added, it's the only required thing for formik

the `inputProps` can override a bunch of defaults or add additional things that should end up on the underlying `<input>` element and formik will make sure to hook up the `onChange`, `onFocus` and `onBlur` events and pass the `value` prop from its own internal form state.

This should also alleviate the need for the `ref`